gyorplusz.js

```diff
@@ -0,0 +1,150 @@
+{
+	"translatorID": "67b93a38-5182-45e5-ab35-3b95764f6979",
+	"label": "Győr Plusz",
+	"creator": "Gemini",
+	"target": "^https?://(www\\.)?gyorplusz\\.hu",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2024-05-24 12:00:00"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2024 Gemini
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (doc.querySelector('div.content-header h1')) {
+		return "magazineArticle";

```

Comment:
This selector doesn't look right - it will return `magazineArticle` if it matches, but it's generic enough, so there's no logical connection with the type being returned. If there any better point to be used for spotting articles?